UPGRADE.md

```diff
@@ -1300,7 +1300,7 @@ The undo feature is partially implemented in the `Notification` component. If yo
 // in src/MyNotification.js
 import * as React from "react";
 import { connect } from 'react-redux';
-import compose from 'recompose/compose';
+import compose from 'recompose/compose'; // You should add recompose/compose to your dependencies
```

Comment:
recompose/compose is a simple compose function.
Not sure this is the case, but if this is the only needed function from recompose we better propose a lighter alternative, like lodash.compose.
Adding the whole recompose library just for the compose function is overkill.